chef-dk/plan.sh

```diff
@@ -0,0 +1,145 @@
+pkg_name=chef-dk
+pkg_origin=core
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="The Chef Developer Kit"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_build_deps=(
+  core/make
+  core/gcc
+  core/coreutils
+  core/git
+)
+
+pkg_deps=(
+  core/glibc
+  core/busybox-static
+  core/ruby
+  core/libxml2
+  core/libxslt
+  core/libiconv
+  core/xz
+  core/zlib
+  core/bundler
+  core/openssl
+  core/cacerts
+  core/libffi
+)
+
+pkg_svc_user=root
```

Comment:
Is this package being run using `hab start core/chef-dk`? It seems not. If not, then this line should be removed.